packages/itwinui-css/src/slider/slider.scss

```diff
@@ -2,116 +2,175 @@
 // See LICENSE.md in the project root for license terms and full copyright notice.
 @use '../mixins';
 
-$thumb-height: var(--iui-size-m); // 16px
-$track-height: var(--iui-size-2xs); // 4px
-$tick-height: var(--iui-size-s); // 12px
+// #region Slider container
+.iui-slider-container {
+  --_iui-slider-container-gap: var(--iui-size-s);
+  --_iui-slider-rail-color: var(--iui-color-border);
+  --_iui-slider-thumb-border-color: var(--iui-color-border-foreground);
+  --_iui-slider-thumb-color: var(--iui-color-background);
+  --_iui-slider-thumb-cursor: grab;
+  --_iui-slider-thumb-size: var(--iui-size-m);
+  --_iui-slider-ticks-placement: 37px; // TODO: Figure out a calc for this
```

Comment:
There's got to be some math we can do here to achieve this, I haven't figured it out yet tho.

Reply:
why does it need this strange margin value anyway? any way to avoid it?

Reply:
At first I tried `.iui-slider-container` having 2 rows & 3 columns.
![Screenshot 2023-09-07 at 1 12 40 PM](https://github.com/iTwin/iTwinUI/assets/849817/618c57be-8019-4bca-8952-ebe83953e6d8)

For the horizontal, I thought I could add a negative top margin to the ticks to get them closer to the rail.  That might work on horizontal, but on vertical we have no idea how wide the 1st column will be, so we can't safely use a negative margin.
![Screenshot 2023-09-07 at 1 14 47 PM](https://github.com/iTwin/iTwinUI/assets/849817/8b5ca47f-5a1f-4098-863e-f6b32d2716f7)

So then I switched `.iui-slider-container` to have 1 row & 3 columns.
![Screenshot 2023-09-07 at 1 06 30 PM](https://github.com/iTwin/iTwinUI/assets/849817/b6c2a32b-a86c-45df-8176-018dd951d760)

And the ticks are placed in the same grid area as the slider itself.
![Screenshot 2023-09-07 at 1 17 06 PM](https://github.com/iTwin/iTwinUI/assets/849817/dea615bb-b934-4d4a-8d80-175397ee735b)

`37px` gets the ticks aligned, but does introduce empty space at the top.
![Screenshot 2023-09-07 at 1 18 36 PM](https://github.com/iTwin/iTwinUI/assets/849817/59df1e6c-2ffc-4694-bdd2-8eda43e8c1f6)
![Screenshot 2023-09-07 at 1 19 50 PM](https://github.com/iTwin/iTwinUI/assets/849817/f2281bee-2f31-4628-b5e8-c3cf23f83980)

It even works when `--_iui-slider-thumb-size`'s value is changed.
![Screenshot 2023-09-07 at 1 20 32 PM](https://github.com/iTwin/iTwinUI/assets/849817/740992e0-6e03-4b10-92c6-8891a3d72934)

I just can't seem to figure out why that is the magic value that works.

Reply:
Also considered doing a grid within a grid, but the min / max grid areas would not align with the slider, it would align with the slider + ticks as a whole.
![Screenshot 2023-09-07 at 1 38 28 PM](https://github.com/iTwin/iTwinUI/assets/849817/91fcc096-f329-4db7-85d4-f1eeedecc298)

Reply:
i think i have an idea for a slight improvement. can you resolve merge conflicts in your branch and then i can take a look?

Reply:
Merge conflicts are resolved.  I'm very curious what your idea is because I have run out of ideas.

Reply:
instead of the margin, you can use `transform`

horizontal:
```
transform: translateY(50%);
```
vertical:
```
transform: translateX(50%);
```

this is exactly the calculation we want, because without the ticks margins, the grid is as big as the slider and we want to place the ticks right in the center.

Reply:
That did it, great suggestion!